src/hooks/index.ts

```diff
@@ -34,6 +34,7 @@ export { default as useTitle } from './useTitle';
 export { default as useTokenInfo, TokenInfoProvider } from './useTokenInfo';
 export { default as useUserAvatarImageFromIPFS } from './useUserAvatarImageFromIPFS';
 export { default as useUserSettings, SlotKey } from './useUserSettings';
+export { default as useCanEditProfile } from './useCanEditProfile';
```

Comment:
Order matters here, needs to go after './useUserSettings' so `SlotKey` is defined